sqlconnect/internal/databricks/mappings.go

```diff
@@ -0,0 +1,84 @@
+package databricks
+
+import (
+	"encoding/json"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/rudderlabs/sqlconnect-go/sqlconnect/internal/base"
+)
+
+// mapping of database column types to rudder types
+var columnTypeMappings = map[string]string{
+	"DECIMAL": "float", // DECIMAL and aliases
+	"NUMERIC": "float",
+	"DEC":     "float",
+
+	"INT":     "int", // INT and aliases
+	"INTEGER": "int",
+
+	"BIGINT": "int", // BIGINT and aliases
+	"LONG":   "int",
+
+	"SMALLINT": "int", // SMALLINT and aliases
+	"SHORT":    "int",
+
+	"TINYINT": "int", // TINYINT and aliases
+	"BYTE":    "int",
+
+	"FLOAT": "float", // FLOAT and aliases
+	"REAL":  "float",
+
+	"DOUBLE":        "float",
+	"BOOLEAN":       "boolean",
+	"STRING":        "string",
+	"CHAR":          "string",
+	"VARCHAR":       "string",
+	"BINARY":        "string",
+	"DATE":          "datetime",
+	"INTERVAL":      "datetime",
+	"VOID":          "string",
+	"TIMESTAMP":     "datetime",
```

Comment:
VOID datatype is interesting.
https://docs.databricks.com/en/sql/language-manual/data-types/null-type.html